docs/template_plugin/backend/opset_int_tbl.hpp

```diff
@@ -128,3 +128,6 @@ NGRAPH_OP(Exp, op::v0)
 NGRAPH_OP(Log, op::v0)
 NGRAPH_OP(PriorBox, ngraph::op::v8)
 NGRAPH_OP(PRelu, op::v0)
+
+NGRAPH_OP(IRDFT, op::v9)
```

Comment:
Why do we need to modify template in this PR? We don't have reference implementation

Reply:
This reference implementation is in the PR https://github.com/openvinotoolkit/openvino/pull/10514.

Reply:
In this case please remove these changes from this PR and it to PR #10514

Reply:
Done.